ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -98,9 +108,13 @@ class SqlLedgerSpec
         )
 
       } yield {
-        ledger1.ledgerId should not be equal(LedgerId)
+        ledger1.ledgerId shouldBe a[LedgerId]
         ledger1.ledgerId shouldEqual ledger2.ledgerId
         ledger2.ledgerId shouldEqual ledger3.ledgerId
+
+        ledger1.participantId shouldBe participantId
```

Comment:
Is this a single participant connected to three ledgers? :thinking:

Reply:
No, this is testing that the ledger id and participant id can be reused after a reset.